### PR TITLE
Bug 1812834: schedule Thanos Querier on worker nodes, make resources configurable

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @openshift/openshift-team-monitoring

### CODE REVIEW

File: pkg/manifests/manifests.go [hunk 1]
```diff
@@ -2508,6 +2508,22 @@ func (f *Factory) ThanosQuerierDeployment(grpcTLS *v1.Secret, enableUserWorkload
 		d.Spec.Template.Spec.Volumes = append(d.Spec.Template.Spec.Volumes, volume)
 	}
 
+	if f.config.ThanosQuerierConfig.Resources != nil {
```
Comment (lilic) [inline pkg/manifests/manifests.go]: So we just default to worker nodes?

**Comment by s-urbaniak** [comment]
/hold
@lilic indeed, i was just about to add a test

File: pkg/manifests/manifests.go (hunk 1 above)
Comment (s-urbaniak) [reply pkg/manifests/manifests.go]: correct, that is what i synced with @brancz

**Comment by s-urbaniak** [comment]
/hold cancel
@lilic ptal

**Pull request merged**